Makefile

```diff
@@ -9,6 +9,9 @@ docs-serve:
 docs: docs-clean
 	sphinx-build -M html docs docs/_build/ -E -a -j auto --keep-going
 
+docs-lint:
+	@vale README.md docs
```

Comment:
Best to be able to execute via pre-commit -

`pre-commit run vale --all-files`

Reply:
This is only temporary & I believe the right way to move forward with QA for the docs is using a pre-commit hook as well. And as I discussed about it earlier as well, I'm waiting for an official response from Vale's author w.r.t to supporting a pre-commit hook themselves. If they're not keen on it then I'll have to rewrite the one I personally maintain.

Reply:
This concern will be resolved after this PR https://github.com/errata-ai/vale/pull/558 is merged.